src/main/resources/site/parts/searchResult/searchResultView.tsx

```diff
@@ -417,8 +396,7 @@ function SearchResult(props: SearchResultProps) {
 
   function renderNameResult() {
     const mainNameResult = props.nameSearchData
-    if (mainNameResult && mainNameResult.count && !filterChanged && numberChanged === 0) {
```

Comment:
If setningen må være som før, kun fjerne addGtagForEvent

Reply:
Changed Back and only removed addGtagForEvent